frontend/src/pages/AboutPage/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Link, useHistory } from "react-router-dom";
+import Button from "../../components/Button";
+import styles from "./style.module.css";
+
+const AboutPage = () => {
+  const browserHistory = useHistory();
+
+  return (
+    <div className={styles.container}>
+      <h1>About</h1>
+      <p className={styles.text}>
+        Nerds Oppose Society is a fun online game that just happens to also be a
+        clone of{" "}
+        <a
+          href="https://cardsagainsthumanity.com/"
+          rel="noreferrer"
+          target="_blank"
+        >
+          Cards Against Humanity®
+        </a>
+        .
+      </p>
+      <p className={styles.text}>
+        Nerds Opposite Society was built over two weeks by the following
+        over-worked University of Auckland students for a Software Engineering
+        course project (SOFTENG 750):
+      </p>
+      <ul>
+        <li>Di Kun Ong (design, frontend)</li>
+        <li>Alexander Verkerk (backend, CI/CD)</li>
+        <li>Rawiri Hohepa (frontend)</li>
+        <li>Tait Fuller (backend)</li>
+      </ul>
```

Comment:
Actually I'd order named alphabetically too

Reply:
Might double check later if we are allowed to link our personal GitHub pages 🤔

Reply:
ADRT just doesn't sound as cool :(

Reply:
Ahhhh well in that case bold our initials please 🤪

Reply:
The specific line on their terms is "no Kickstarter projects, no ads or donations accepted on your website, no links to your Patreon thing, no distribution through app stores, no using it to promote your brand or your cause, etc." Hmm

Reply:
Hmmmm yeah something to have a think about down the line for our CVs - won't worry about it for now